tests/test-backend-ops.cpp

```diff
@@ -5940,6 +5941,7 @@ static std::vector<std::unique_ptr<test_case>> make_test_cases_eval() {
 
     test_cases.emplace_back(new test_count_equal(GGML_TYPE_F32, {4,  500, 1, 1}));
     test_cases.emplace_back(new test_count_equal(GGML_TYPE_F32, {4, 5000, 1, 1}));
+    test_cases.emplace_back(new test_count_equal(GGML_TYPE_I32, { 512, 16, 1, 1}));
```

Comment:
Changing the type of `test_count_equal` will do nothing. There is no point to adding this test.